src/app/common/if-not-null.pipe.ts

```diff
@@ -12,10 +12,15 @@ export class IfNotNullPipe implements PipeTransform {
     const suffix = args[2] || '';
 
     let property = null;
-    for (const prop of properties.split('.')) {
-      property = value[prop];
-      value = property;
+    try {
+      for (const prop of properties.split('.')) {
+        property = value[prop];
+        value = property;
+      }
+    } catch {
+      return '--';
```

Comment:
Try catch implemented to avoid TypeErrors when the chaining is long and we are not sure if the value contains all of the keys.